include/pisa/wand_data_range.hpp

```diff
@@ -32,40 +32,41 @@ class wand_data_range {
 
     class builder {
        public:
-        builder(partition_type                type,
+        builder(partition_type type,
```

Comment:
The `[[maybe_unused]]` attribute is supposed to tell the compiler that this argument might not be used, and therefore it won't generate a warning.

The same thing was achieved before with `(void)params;`. Now, this should be removed.